src/hooks/useLogout.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from 'react';
+import { logout } from '@multiversx/sdk-dapp/utils/logout';
+
+export const useLogout = () => {
+  const clearStorage = useCallback(() => {
+    localStorage.clear();
+    sessionStorage.clear();
+  }, []);
+
+  return useCallback(
+    async (
+      callbackUrl?: string,
```

Comment:
more than 2 parameters should go into an object

Reply:
Kept the same signature as the login function in `sdk-dapp`. Just for consistency